src/App.tsx

```diff
@@ -1,8 +1,17 @@
 import React from "react";
 import "./App.css";
+import { BrowserRouter, Routes, Route, Navigate } from "react-router-dom";
+import Products from "./Pages/ProductsPage";
 
 function App() {
-  return <div className="App">test</div>;
+  return (
+    <BrowserRouter>
+      <Routes>
+        <Route path="/products" element={<Products />} />
```

Comment:
Create URL constants file 
```suggestion
        <Route path={URL.PRODUCTS} element={<Products />} />
```